config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
+      resources :items, except: [:new, :edit]
```

Comment:
Looks good. still need to add routes for the additional actions in the controller